src/main/ui/app/component/AddOn.jsx

```diff
@@ -1,30 +1,42 @@
 import {Component} from "react";
 import {Link} from "react-router";
+import { ListGroupItem , Badge, Accordion , Panel , Media , Button } from 'react-bootstrap';
 
 export default class AddOn extends Component {
 
     render() {
         const addon = this.props.addon;
         const icon = addon.type === 'OWA' ? 'fa fa-2x fa-globe' : 'fa fa-2x fa-puzzle-piece';
-        const link = this.props.version ? `/show/${addon.uid}?highlightVersion=${this.props.version}` : `/show/${addon.uid}`;
+        const link = this.props.version ? `#/show/${addon.uid}?highlightVersion=${this.props.version}` : `#/show/${addon.uid}`;
+	
+	const title = (
+ <div>{addon.name} <div className="takeright">{addon.type}</div></div>
+);
+
         return (
-                <Link to={link}>
-                    <div className="media panel">
-                        <div className="media-left">
+                
+                    <ListGroupItem className="mainlist">
+                   <Media>
+                        <Media.Left>
                             <i className={icon} aria-hidden="true"></i>
-                        </div>
-                        <div className="media-body">
-                            <h4 className="media-heading">
-                                {addon.name}
-                                <span className="badge">
-                                { this.props.version || "" }
-                            </span>
-                            </h4>
-                            {addon.description}
-                        </div>
-                    </div>
-                </Link>
+                     	</Media.Left>
+                   <Media.Body> 
+                            
+    			
+		            <Accordion>
```

Comment:
I don't like the accordion effect here. Just put the description text directly. (Maybe later we'll need to do something to shorten the text if it's too long.)